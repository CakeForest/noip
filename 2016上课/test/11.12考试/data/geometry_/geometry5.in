10
48 24 8 95 83 64 52 30 61 74 
12 30 76 1 70 4 40 80 80 10 
10
80 60
25 88
40 32
15 65
70 53
20 13
42 22
86 4
0 0
48 80
