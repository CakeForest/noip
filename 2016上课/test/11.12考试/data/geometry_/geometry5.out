10
10
7
8
10
4
6
9
0
10
